examples/account/config.rs

```diff
@@ -46,8 +46,7 @@ async fn main() -> Result<()> {
       // Configure a client for the private network
       ClientBuilder::new()
         .network(network.clone())
-        .primary_node(private_node_url, None, None)?
-        // .permanode(<permanode_url>, None, None)? // set a permanode for the same network
+        .primary_node(private_node_url, None, None)?, // .permanode(<permanode_url>, None, None)? // set a permanode for the same network
```

Comment:
```suggestion
        .primary_node(private_node_url, None, None)?,
         // set a permanode for the same network
         // .permanode(<permanode_url>, None, None)?
```

I think it should be this :laughing: Putting everything in one line is very unreadable, imo.